pkg/realtime/metrics.go

```diff
@@ -421,3 +434,13 @@ func getName(c prometheus.Collector) string {
 	}
 	return ""
 }
+
+func addLabel(ls []string, l string) []string {
+	ns := make([]string, len(ls))
+
+	copy(ns, ls)
+
+	ns = append(ns, l)
+
+	return ns
+}
```

Comment:
In the other subsystems, I *think* we have cases were we'll need to add more than one label to the label set. So this might be more reusable as a variadic func:

```suggestion
func addLabel(ls []string, l ...string) []string {
	ns := make([]string, len(ls), len(ls)+len(l))
	copy(ns, ls)

	return append(ns, l...)
}
```

Reply:
You will see this change applied in pull request #167. 👍